app/views/shared/_header.html.slim

```diff
@@ -1,13 +1,13 @@
 javascript:
-  if (! !!location.hostname.includes('globalforestwatch.org'))
+  if (location.hostname.indexOf('globalforestwatch.org') !== -1)
```

Comment:
Sorry, I didn't explain correctly. It should be on every place but PRO. So the condition should change to == -1